autopush/web/webpush.py

```diff
@@ -78,6 +78,14 @@ def validate_uaid_month_and_chid(self, d):
         if result.get("router_type") not in ["webpush", "gcm", "apns", "fcm"]:
             raise InvalidRequest("Wrong URL for user", errno=108)
 
+        if result.get("router_type") in ["gcm", "fcm"]:
+            if 'senderID' not in result.get('router_data',
```

Comment:
because I broke them apart when I was fixing the tests and forgot to glue them back together?

me === idiot.